src/core/crypto/privateKey.ts

```diff
@@ -27,3 +30,91 @@ export interface PrivateKey {
    */
   toUint8Array(): Uint8Array;
 }
+
+export class PrivateKey {
+  /**
+   * Returns the AIP-80 compliant prefix for a given private key type. Append this to a PrivateKey HexString
+   * to get an AIP-80 compliant string.
+   *
+   * [Read about AIP-80](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-80.md)
+   *
+   * @param type
+   * @returns
+   */
+  public static getAIP80Prefix(type: "ed25519" | "secp256k1"): string {
```

Comment:
we should define a type for the argument. 
do we accept the other types? https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/types/index.ts#L105